### DISCUSSION TITLE
some feature request for Iw3

### DISCUSSION THREAD

**Discussion by loawizard** [nagadomi/nunif#82] [General]
Thank you so much for the amazing software.

Here are some feature requests that could make the software even better:
-Resume button.
-only right eye render/only left eye render: in this feature, you only render an image for one eye, and the other eye stays like the original output. This could reduce artifacts like noses being clipped 
-Subtitle support
-mkv support: mkv makes it possible for you to watch the videos even if it's exporting . I have seen this work in an inferior 3D creation product. Would be amazing if that would be possible.

Iw3 is Truly amazing software!
Thank you so much for creating it.

**Comment by nagadomi** [comment]
>resume

I will try to support this at some point, but it will be a partial implementation that will fail under some conditions.
For it to work perfectly, each frame image should be saved, but that causes processing speed to slow down.

>only right eye render/only left eye render

This method produces more artifacts in the image of one eye than the current setting, so the overall result may be worse.

>Subtitle support

This can be easily addressed.
However, I don't know whether VR video players supports subtitling for SBS video.
It is possible to render subtitles, but I believe that feature should be supported by the video player.

>mkv support

mkv is already supported. You can specify the output file name for `*.mkv` directly(e. g. `/output_folder/filename_LRF.mkv`). However, I have not tested it much.

**Comment by loawizard** [comment]
MKV works brilliantly!  I can watch it while converting now !

thank you for looking into a resume feature that does'nt slow down the processing
